functional/OpenJcePlusTests/playlist.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
+	<test>
+		<testCaseName>openJcePlusTests</testCaseName>
+		<disables>
+			<disable>
+				<comment>only applicable for Semeru atm</comment>
+				<vendor>eclipse</vendor>
```

Comment:
When testing a custom built openj9 build with `OpenJCEPlus` bundling turned on for development builds ( not the default for public nightly openj9 builds ) would these tests still be able to be run? I believe the answer is yes yet I am asking anyways due to this disablement of the tests when the vendor is `eclipse`.

Reply:
Yes, custom built openj9 build with OpenJCEPlus bundling can still run by setting TEST_FLAG=OpenJcePlusTests (for test compilation) and TARGET=disabled.openJcePlusTests (for test execution)